include/engine/datafacade/internal_datafacade.hpp

```diff
@@ -587,6 +587,11 @@ class InternalDataFacade final : public BaseDataFacade
         return result;
     }
 
+    std::string GetPronunciationForID(const unsigned name_id) const override final
+    {
+        return GetNameForID(name_id + 1);
```

Comment:
Doing the subsequent copies into the char data and using the name_id as base here is _really_ clever --- we should document that somewhere!